src/class/object_genome_variants.php

```diff
@@ -222,20 +222,6 @@ function __construct ()
 
 
 
-    function buildForm ($sPrefix = '')
-    {
-        $aForm = parent::buildForm($sPrefix);
-        // Link to HGVS for nomenclature.
-        if (isset($aForm[$sPrefix . 'VariantOnGenome/DNA'])) {
-            $aForm[$sPrefix . 'VariantOnGenome/DNA'][0] = str_replace('(HGVS format)', '(<A href="http://varnomen.hgvs.org/recommendations/DNA" target="_blank">HGVS format</A>)', $aForm[$sPrefix . 'VariantOnGenome/DNA'][0]);
-        }
-        return $aForm;
-    }
-
-
-
-
-
```

Comment:
@ifokkema, this was removed in October (see [commit](https://github.com/LOVDnl/LOVD3/commit/1d3dad73e57a6ba4adeea040b616ef0c8083eaee)). Do you remember why this was deleted?